src/interface/web/app/components/chatMessage/chatMessage.tsx

```diff
@@ -554,6 +574,7 @@ const ChatMessage = forwardRef<HTMLDivElement, ChatMessageProps>((props, ref) =>
                     className={styles.chatMessage}
                     dangerouslySetInnerHTML={{ __html: markdownRendered }}
                 />
+                {excalidrawData && <ExcalidrawComponent data={excalidrawData} />}
```

Comment:
Can we make it such that the `ExcalidrawComponent` only captures scroll when clicked? (Or disable scroll in excalidraw block when previous scroll event was outside the excalidraw window and <1 second ago). 

Currently when scrolling through chat history excalidraw diagrams in it capture the scroll when scrolling past them. This is unexpected and annoying.

Reply:
Just updating to say I haven't been able to workaround this issue. I suggest we merge it now and loop back to it after the release.